Chapter-4-applying-tactical-domain-driven-design/Fitnet.Contracts/Src/Fitnet.Contracts.IntegrationTests/SignContract/SignContractRequestParameters.cs

```diff
@@ -2,17 +2,17 @@ namespace EvolutionaryArchitecture.Fitnet.Contracts.IntegrationTests.SignContrac
 
 using Api;
 
-internal record SignContractRequestParameters(string Url, DateTimeOffset SignedAt)
+internal record SignContractRequestParameters(string Url, DateTimeOffset SignedAt, string SignatureText)
```

Comment:
Btw. I would just name it Signature,not SignatureText but this is my personal opinion. What do you think?